ssl/ssltest.c

```diff
@@ -371,6 +379,20 @@ static int verify_ticket(SSL* ssl)
     return -1;
 }
 
+static int verify_server_digest(SSL* ssl)
+{
+    int nid = NID_undef;
+
+    if (server_digest_expect == NULL)
+        return 0;
```

Comment:
nit: the tristate return is never used, so you could just return 1 on success/no expectation, and 0 on failure?

Reply:
That would mean also changing the `verify_*` calls since the two calling conventions are inconsistent. (0 is a no-expectation-configured success, not failure, here.) It seemed weird to make it different from `verify_{alpn,servername,ticket}`. Thoughts?

Reply:
Yeah, I suspected you might have done it for consistency. Let's leave it alone then, in 1.1.* this test is deprecated anyway.